src/bin.js

```diff
@@ -22,6 +22,7 @@ program
   .option("-b, --browsers <items>", "browsers list (comma separated)")
   .option("-I, --no-import", "do not inline @import")
   .option("-U, --no-url", "do not adjust url()")
+  .option("-e, --extract", "output values of custom properties")
```

Comment:
Good idea. But since this extracts from imported files also, do you think we should omit the "in a file" part?

And since currently only properties/media are extracted, and there is no indication any more types of "variables" will be specced. How about we just say "extract resolved custom properties and custom medias"? We can always expand that later. Also do you prefer "custom media" or "custom medias"? This phrase also appears in the extracted json object.